initrd/bin/kexec-insert-key

```diff
@@ -51,6 +51,8 @@ tpm extend -ix 4 -ic generic \
 
 # Check to continue
 if [ "$unseal_failed" = "y" ]; then
+	diff "$(dirname $INITRD)/kexec_lukshdr_hash.txt" /tmp/luksDump.txt \
```

Comment:
@root-hardenedvault : wouldn't be preferable to inform the user if the hashes differ? (! diff)